### PR TITLE
Issue #22: Provides a verbose option that prevents bee from outputting warnings…

### LINKED ISSUE #22: Add a --quiet option for bee commands.
What I had in mind was something like the `if($type == 'log' && !b_is_debug())` filter in `includes/common.inc` (around line 84) that would similarly filter out status, success and log messages.

The goal is to have the `--quiet`option either suppress all output (like drush) and note success or failure through the return value in $?, or they reduce the output, removing the progress and status messages, but leaving any warnings or errors. Some commands (mplayer, for example) support multiple levels of quiet, like multiple levels of -v for verbosity.

For example using `b` to trigger cron is a common use case.  The current version of `b` will print out a success message if everything works, which when run through cron creates a lot of emails.

We should add a `--quiet` (`-q`) option that doesn't print out any of the success or log messages, just the errors and warnings.  Perhaps two `-q` options could suppress the warnings, too.

### PR DESCRIPTION
**Pull request by rbargerhuff**
Fixes #22

### CODE REVIEW

File: includes/globals.inc [hunk 1]
```diff
@@ -51,12 +55,15 @@ $_bee_backdrop_site = FALSE;
...
 
-// A boolean indicating whether or not `Yes` mode is enabled.
+// A boolean indicating whether `Yes` mode is enabled.
 $_bee_yes_mode = FALSE;
 
-// A boolean indicating whether or not `Debug` mode is enabled.
+// A boolean indicating whether `Debug` mode is enabled.
 $_bee_debug_mode = FALSE;
 
+// A boolean indicating whether 'verbose' mode is enabled.
```
Comment (ghost) [inline includes/globals.inc]: Best to copy syntax from other examples above:

```suggestion
// A boolean indicating whether `Verbose` mode is enabled.
```

**Review by ghost — CHANGES_REQUESTED** [review]
One very small fix.

**Comment by rbargerhuff** [comment]
My collegue and I who originally made this Pull Request noticed that the change made by @BWPanda had not been commited to the main branch and we are wondering how this request can be included.

**Review by rbargerhuff — COMMENTED** [review]
This change is great. Thank you!

**Comment by yorkshire-pudding** [comment]
Hi @rbargerhuff - I'm sorry this wasn't merged earlier.  Would you be willing to to a rebase of your PR so that it can be merged as it is currently causing a conflict due to recent changes for PHP 8.1 compatibility?
In particular, this line in `bee.php`
```php
function bee_error_handler($errno, $message, $filename, $line, $context) {
```
has changed to
```php
function bee_error_handler($errno, $message, $filename, $line, $context = NULL) {
```

And because your branch is based on an older version, there is a conflict there.
This looks like a great addition so would be great to get it in.

**Comment by rbargerhuff** [comment]
Abandoning this PR since it is no longer applicable.

**Pull request closed without merging**